src/test/scala/org/scalacheck/magnolia/test/MagnoliaTest.scala

```diff
@@ -51,25 +51,31 @@ object MagnoliaTest extends TestSuite {
       p3 <- expectedTestTraitGen
     } yield Recursive(p1, p2, p3)
 
+  locally {
+    import org.scalacheck.Cogen
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/manual/ChocPanda/scalacheck-magnolia/pullRequest?prid=4177247)